typescript.json

```diff
@@ -0,0 +1,88 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "commitMessageAction": "bump",
+  "dependencyDashboard": true,
+  "dependencyDashboardAutoclose": true,
+  "enabledManagers": ["npm", "github-actions", "dockerfile"],
+  "extends": [
+    ":approveMajorUpdates",
+    ":automergeAll",
+    ":combinePatchMinorReleases",
+    ":prConcurrentLimit10",
```

Comment:
hm, so you can't have more than 10 parallel PRs open, but renovate also (`:prHourlyLimit4`) will not create more than 4 PRs per hour in span of 2 hours?

Reply:
That is what I think, indeed. Although due to grouping is unlikely to get more than 3 or 4 PRs.